src/app-constants.js

```diff
@@ -43,6 +43,7 @@ const requiredEnvVars = [
   'PRODUCT_PROMOS_ENABLED',
   'REDIS_URL',
   'SENTRY_DSN',
+  'SENTRY_DSN_LEGACY',
```

Comment:
Is this actually a required var, or would it work as "optional"?  

I think a missing "required" var break/blocks the site, but optional allows it to run, in the event a dev might not have the key.

If this is for the legacy site, do we even need to add it to the `src` constants?